recipes/m2crypto/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.23.0" %}
+
+package:
+  name: m2crypto
+  version: {{ version }}
+
+source:
+  fn: M2Crypto-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/M/M2Crypto/M2Crypto-{{ version }}.tar.gz
+  md5: 89557730e245294a6cab06de8ad4fb42
+
+  patches:
+    - setup.patch
```

Comment:
Sorry, I should have been more clear. While the patch makes sense, we should explain in a comment above the patch file why it was added. Your response to my question would be adequate.

As far as patches go, it is our general rule that they must be submitted (not necessarily accepted yet) upstream. So, even though this is simple and makes sense, we still want to get the ball rolling. We can then reference the PR, mailing list entry, etc. above the patch. The goal in general is that we don't find ourselves maintaining tons of patches, but that we are pushing them back into the source so that we should no longer need to maintain them.

I hope that makes more sense.